src/stmt.rs

```diff
@@ -432,6 +439,19 @@ pub struct CTCount {
     pub inv: Option<bool>,
 }
 
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+/// Limit the number of connections using conntrack.
+pub struct SynProxy {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    /// maximum segment size (must match your backend server)
+    pub mss: Option<u32>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    /// window scale (must match your backend server)
+    pub wscale: Option<u32>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub flags: Option<HashSet<SynProxyFlag>>,
```

Comment:
There could be another doc comment added here.

Reply:
Thanks, fixed in 855b72b8dfb9a3010a84c71c290a724e9a75743c